### PR TITLE
add resources html

### PR DESCRIPTION
**Pull request by vikumkbv**

### CODE REVIEW

File: src/main/resources/templates/parts/toptiles.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+            <span class="count_top "><i class="fa fa-graduation-cap"></i>  Modules Enrolled</span>
+            <div class="count" th:text="${topTiles.modules}"></div>
+        </div>
+    </div>
+    </div>
+    <!-- top tiles end -->
+
+</th:block>
+
+</html>
```
Comment (gigara) [inline src/main/resources/templates/parts/toptiles.html:23]: Add new line

File: src/main/resources/templates/parts/topmenu.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+                    </li>
+                </ul>
+            </nav>
+        </div>
+    </div>
+    <!-- /top navigation -->
+
+</th:block>
+
+</html>
```
Comment (gigara) [inline src/main/resources/templates/parts/topmenu.html:35]: add new line

File: src/main/resources/templates/parts/styles.html [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+<th:block th:fragment="datatable">
+    <!-- Datatables -->
+    <link th:href="@{/vendors/datatables.net-bs/css/dataTables.bootstrap.min.css}" rel="stylesheet">
+    <link th:href="@{/vendors/datatables.net-buttons-bs/css/buttons.bootstrap.min.css}" rel="stylesheet">
+    <link th:href="@{/vendors/datatables.net-fixedheader-bs/css/fixedHeader.bootstrap.min.css}" rel="stylesheet">
+    <link th:href="@{/vendors/datatables.net-responsive-bs/css/responsive.bootstrap.min.css}" rel="stylesheet">
+    <link th:href="@{/vendors/datatables.net-scroller-bs/css/scroller.bootstrap.min.css}" rel="stylesheet">
+</th:block>
+
+</html>
```
Comment (gigara) [inline src/main/resources/templates/parts/styles.html:26]: Add new line

File: src/main/resources/templates/parts/sidemenu.html [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+                    </ul>
+                </div>
+
+            </div>
+            <!-- /sidebar menu -->
+        </div>
+    </div>
+</th:block>
+
+</html>
```
Comment (gigara) [inline src/main/resources/templates/parts/sidemenu.html:50]: add new line

File: src/main/resources/templates/parts/footer.html [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+<th:block th:fragment="footer">
+    <!-- footer content -->
+    <footer>
+        <div class="pull-right">BlackBoard <a href="https://gigara.info">Giga</a> - 2020</div>
+        <div class="clearfix"></div>
+    </footer>
+    <!-- /footer content -->
+</th:block>
+
+</html>
```
Comment (gigara) [inline src/main/resources/templates/parts/footer.html:14]: add new line

File: src/main/resources/templates/member/list.html [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+        <th:block th:replace="parts/footer :: footer"></th:block>
+
+    </div>
+</div>
+
+<th:block th:replace="parts/scripts :: common"></th:block>
+<th:block th:replace="parts/scripts :: datatable"></th:block>
+<script th:src="@{/custom/js/book/list.js}"></script>
+</body>
+</html>
```
Comment (gigara) [inline src/main/resources/templates/member/list.html:69]: add new line

File: src/main/resources/templates/member/add.html [hunk 7]
```diff
@@ -0,0 +1,141 @@
...
+
+    </div>
+</div>
+
+<th:block th:replace="parts/scripts :: common"></th:block>
+<!-- jquery.inputmask -->
+<script th:src="@{/vendors/jquery.inputmask/dist/min/jquery.inputmask.bundle.min.js}"></script>
+<script th:src="@{/custom/js/member/form.js}"></script>
+</body>
+</html>
```
Comment (gigara) [inline src/main/resources/templates/member/add.html:141]: add new line

File: src/main/resources/templates/login.html [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+                        <h1><i class="fa fa-pencil"></i> BlackBoard Giga</h1>
+                    </div>
+                </div>
+            </form>
+        </section>
+    </div>
+</div>
+
+</body>
+</html>
```
Comment (gigara) [inline src/main/resources/templates/login.html:49]: add new line

File: src/main/resources/templates/home.html [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+        <!-- /page content -->
+
+        <th:block th:replace="parts/footer :: footer"></th:block>
+
+    </div>
+</div>
+
+<th:block th:replace="parts/scripts :: common"></th:block>
+</body>
+</html>
```
Comment (gigara) [inline src/main/resources/templates/home.html:37]: add new line

File: src/main/resources/templates/coursework/view.html [hunk 10]
```diff
@@ -0,0 +1,79 @@
...
+        <th:block th:replace="parts/footer :: footer"></th:block>
+
+    </div>
+</div>
+
+<th:block th:replace="parts/scripts :: common"></th:block>
+<th:block th:replace="parts/scripts :: datatable"></th:block>
+<script th:src="@{/custom/js/book/list.js}"></script>
+</body>
+</html>
```
Comment (gigara) [inline src/main/resources/templates/coursework/view.html:79]: add new line

File: src/main/resources/templates/coursework/submit.html [hunk 11]
```diff
@@ -0,0 +1,89 @@
...
+
+        <th:block th:replace="parts/footer :: footer"></th:block>
+
+    </div>
+</div>
+
+<th:block th:replace="parts/scripts :: common"></th:block>
+<script th:src="@{/custom/js/book/form.js}"></script>
+</body>
+</html>
```
Comment (gigara) [inline src/main/resources/templates/coursework/submit.html:89]: add new line

File: src/main/resources/templates/coursework/add.html [hunk 12]
```diff
@@ -0,0 +1,108 @@
...
+
+        <th:block th:replace="parts/footer :: footer"></th:block>
+
+    </div>
+</div>
+
+<th:block th:replace="parts/scripts :: common"></th:block>
+<script th:src="@{/custom/js/book/form.js}"></script>
+</body>
+</html>
```
Comment (gigara) [inline src/main/resources/templates/coursework/add.html:108]: add new line

File: src/main/resources/application.properties [hunk 13]
```diff
@@ -0,0 +1,10 @@
+
+spring.datasource.url = jdbc:mysql://localhost:3306/BBGiga
+spring.datasource.username = root
+spring.datasource.password =
+
+spring.jpa.hibernate.ddl-auto = update
+spring.jpa.properties.hibernate.dialect = org.hibernate.dialect.MySQL5Dialect
+spring.queries.users-query=select username, password, active from user where username=?
+spring.queries.roles-query=select username, role from user where username=?
+spring.datasource.initialization-mode=always
```
Comment (gigara) [inline src/main/resources/application.properties:10]: add new line

**Comment by vikumkbv** [comment]
as all the suggestion are new line and has no much effect I'll merge this

**Pull request merged**